### DISCUSSION TITLE
recorded videos

### DISCUSSION THREAD

**Discussion by KrishnaraoShinde** [akash-coded/mern#36] [Q&A]
@akash-coded 
where will we find, the text you type in the comment/chat section , while the live class is going on live

@akash-coded

**Comment by santu1667** [comment]
@KrishnaraoShinde   Check this link for recorded videos , i think chat wont be saved u need to save it manually 
https://ecorp.edureka.co/tcstech/my-classroom/mern-training/recordedclasses?courseId=1595&fromMasterCourse=0
